src/utils/listener.js

```diff
@@ -22,21 +40,85 @@ Listener.prototype.destroy = function() {
     this._connection = null;
 };
 
+/*
+ * This function can be called by the provider within the callback function.
+ * Either accept or reject needs to be called, otherwise it prints out a deprecated warning.
+ *
+ * @returns {void}
+ */
+Listener.prototype.accept = function( name ) {
+    this._connection.sendMsg( this._type, C.ACTIONS.LISTEN_ACCEPT, [ this._pattern, name ] );
+    this._responded = true;
+}
+
+/*
```

Comment:
This documentation is a bit vague, better to say something like

` 
Rejecting a listener request informs deepstream that the current provider is not willing 
to provide the  record or event matching the subscriptionName . This will result in deepstream requesting another
provider to do so instead. If no other provider accepts or exists, the record will remain unprovided.
`